pkg/logs/client/http/destination.go

```diff
@@ -117,6 +117,7 @@ func newDestination(endpoint config.Endpoint,
 		policy = backoff.NewConstantBackoffPolicy(
 			coreConfig.Datadog.GetDuration("serverless.constant_backoff_interval"),
 		)
+		shouldRetry = false
```

Comment:
Can you explain this change? I am confused because I thought we wanted the retries.

Reply:
➕ This change as is will disable retries for:
- all of serverless
- the core agent (HTTP logs)
- epforwarder and all of their scnearios

Reply:
The problem with that retry logic is that it will keep retrying forever and keep the extension running till timeout.
I changed the code to place that in a better place and affecting only serverless